connect/api/src/main/java/org/apache/kafka/connect/transforms/Transformation.java

```diff
@@ -33,6 +33,11 @@
      * Apply transformation to the {@code record} and return another record object (which may be {@code record} itself) or {@code null},
      * corresponding to a map or filter operation respectively.
      *
+     * A transformation must not mutate objects reachable from the given {@code record}
+     * (including, but not limited to, {@link org.apache.kafka.connect.header.Headers},
```

Comment:
Should we change `{@link org.apache.kafka.connect.header.Headers}` to `{@link org.apache.kafka.connect.header.Headers Headers}` ?